Classes/Utility/ContentUtility.php

```diff
@@ -0,0 +1,49 @@
+<?php
+declare(strict_types = 1);
+
+namespace FriendsOfTYPO3\Headless\Utility;
+
+/***
+ *
+ * This file is part of the "headless" Extension for TYPO3 CMS.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE.txt file that was distributed with this source code.
+ *
+ *  (c) 2019
+ *
+ ***/
+
+/**
+ * ContentUtility
+ */
+class ContentUtility
+{
+    /**
+     * @param $content
```

Comment:
plz add description

Reply:
All methods in this class now have description